integration/exec_test.go

```diff
@@ -12,6 +12,17 @@ import (
 )
 
 func TestExecPS(t *testing.T) {
+	testExecPS(t, false)
+}
+
+func TestUsernsExecPS(t *testing.T) {
+	if _, err := os.Stat("/proc/self/ns/user"); os.IsNotExist(err) {
```

Comment:
@LK4D4 @mrunalp  userns is a special one, because it was added recently. Many distributives provide kernels where it's disabled.
Actually many people uses containers without userns. And when we enable userns, the containers is created by another way, so we need to test both configurations.